components/infra-proxy-service/server/nodes.go

```diff
@@ -60,6 +60,53 @@ func (s *Server) DeleteNode(ctx context.Context, req *request.DeleteNode) (*resp
 	}, nil
 }
 
+// UpdateNode update the node attributes
+func (s *Server) UpdateNode(ctx context.Context, req *request.UpdateNode) (*response.UpdateNode, error) {
+	c, err := s.createClient(ctx, req.OrgId, req.ServerId)
+	if err != nil {
+		return nil, err
+	}
+
+	automate, err := StructToJSON(req.AutomaticAttributes)
```

Comment:
"automate" is the product name--very confusing. 😁 
```suggestion
	automatic, err := StructToJSON(req.AutomaticAttributes)
```